examples/skywalking-tracing/docker-compose.yaml

```diff
@@ -0,0 +1,112 @@
+version: "3.7"
+services:
+
+  # Front envoy.
+  front-envoy:
+    build:
+      context: .
+      dockerfile: Dockerfile-frontenvoy
+    networks:
+      - envoymesh
+    expose:
+      - "8000"
+      - "8001"
+    ports:
+      - "8000:8000"
+      - "8001:8001"
+    depends_on:
+      - skywalking-oap
+  # First service.
+  service1:
+    build:
+      context: ../front-proxy
+      dockerfile: Dockerfile-service
+    volumes:
+      - ./service1-envoy-skywalking.yaml:/etc/service-envoy.yaml
+    networks:
+      - envoymesh
+    environment:
+      - SERVICE_NAME=1
+    expose:
+      - "8000"
+    depends_on:
+      - skywalking-oap
+  # Second service.
+  service2:
+    build:
+      context: ../front-proxy
+      dockerfile: Dockerfile-service
+    volumes:
+      - ./service2-envoy-skywalking.yaml:/etc/service-envoy.yaml
+    networks:
+      - envoymesh
+    environment:
+      - SERVICE_NAME=2
+    expose:
+      - "8000"
+    depends_on:
+      - skywalking-oap
+  # Skywalking components.
+  elasticsearch:
+    image: elasticsearch:7.9.2
+    container_name: elasticsearch
+    restart: always
```

Comment:
this causes the container to start even when the user reboots. 

i think it should probably have a `restart_policy` of `none`